index.html

```diff
@@ -4045,7 +4033,7 @@ <h2>
       </code></p>
 
       <p>
-The input variables of this function MUST be as follows:
+The input variables of this function are as follows:
```

Comment:
Eliminating this normative statement has no effect on the number of tests we write because each item in the list asserts a normative test as well and refers to a data structure for which there are already normative tests.